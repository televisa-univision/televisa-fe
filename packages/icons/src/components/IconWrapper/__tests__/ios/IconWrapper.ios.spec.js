import shared from '../sharedTest';

/** @test {IconWrapper} */
describe('IconWrapper on ios', () => {
  shared();
});
