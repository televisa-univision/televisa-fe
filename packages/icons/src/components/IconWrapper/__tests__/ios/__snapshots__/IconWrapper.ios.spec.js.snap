// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconWrapper on ios should render correctly 1`] = `
<svg
  data-name="close"
  style={Object {}}
/>
`;

exports[`IconWrapper on ios should render fallback on error 1`] = `<FallbackSvg />`;

exports[`IconWrapper on ios should render without fallback on error 1`] = `""`;
